python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -66,7 +66,15 @@
 from .singleQuantumExecutor import SingleQuantumExecutor
 
 if TYPE_CHECKING:
-    from lsst.daf.butler import DatastoreRecordData, LimitedButler, Quantum, Registry
+    from lsst.daf.butler import (
```

Comment:
We probably should add that coverage exception list from daf_butler pyproject.toml.